main.go

```diff
@@ -76,9 +80,11 @@ func main() {
 
 	run(ctx, cfg)
 
-	for range time.NewTicker(time.Hour * 10).C {
-		run(ctx, cfg)
-	}
+  if viper.GetBool("DAEMON") {
```

Comment:
In order to keep the code more readable, it would be great if the `if viper.GetBool("DAEMON")` conditional statement is merged with the one in line 67